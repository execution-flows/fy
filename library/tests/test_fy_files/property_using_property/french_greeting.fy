property greeting using french_greeting:
    with property french_greeting

    def -> str:
        return self._french_greeting
