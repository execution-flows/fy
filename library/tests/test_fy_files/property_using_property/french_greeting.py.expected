import datetime


import abc

from test_fy_files.property_using_property.abc import With_FrenchGreeting_PropertyMixin_ABC


class Greeting_UsingFrenchGreeting_PropertyMixin(
    # Property_mixins
    With_FrenchGreeting_PropertyMixin_ABC,
    abc.ABC
):
   @property
   def _greeting(self) -> str:
        return self._french_greeting
